### ISSUE TITLE
meuzu  5.1 报错

### ISSUE THREAD

**Issue by zhangle1** [MartinRGB/RapidInterpolator#3]
已炸- -                    
    Caused by: java.lang.ClassCastException: android.graphics.drawable.VectorDrawable cannot be cast to android.graphics.drawable.BitmapDrawable

**Comment by zhangle1** [comment]
魅族 5.1  我放入 svg 转的 xml文件报错  ...

**Comment by zhangle1** [comment]
大神 求支持下svg

**Comment by zhangle1** [comment]
红米4.4note是ok的

**Comment by MartinRGB** [comment]
drawable animation？我是基于 ObjectAnimator拓展的

**Comment by zhangle1** [comment]
不是  就是 app:img  设置图片 如果放的是  svg 转的xml 矢量图 那么在 5.1的魅族手机里会蹦 ，但是放png图片是好的  android 提示我的是Caused by: java.lang.ClassCastException: android.graphics.drawable.VectorDrawable cannot be cast to android.graphics.drawable.BitmapDrawable失败

**Comment by MartinRGB** [comment]
老哥你是魅族的吗
